packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4237,6 +4242,7 @@ export const SendBox: (props: SendBoxProps) => JSX.Element;
 
 // @beta
 export interface SendBoxErrorBarError {
+    errorBarType?: MessageBarType;
```

Comment:
We should use our own type, not one from FluentUI (to be flexible if we need to switch from another version of FluentUI or UI framework)